Python/Tests/TestAdapterTests/TestDiscovererTests.cs

```
@@ -688,7 +689,20 @@ public abstract partial class TestDiscovererTests {
             var discoveryContext = new MockDiscoveryContext(runSettings);
             var discoverySink = new MockTestCaseDiscoverySink();
             var logger = new MockMessageLogger();
-            var discoverer = new PythonTestDiscoverer();
+
+            ITestDiscoverer discoverer = null;
+            switch (testEnv.TestFramework) {
+                case FrameworkPytest:
+                    discoverer = new PytestTestDiscoverer();
+                    break;
+
+                case FrameworkUnittest:
+                    discoverer = new UnitTestDiscoverer();
+                    break;
+                default:
+                    Assert.Fail();
```

should pass in a message to `Fail`, "Unknown test framework"